platform/logging/appenders/file/FileAppenderConfig.ts

```diff
@@ -0,0 +1,33 @@
+import * as schema from '../../../lib/schema';
+import { BaseAppenderConfig, getRawSchema as getBaseRawSchema } from '../base/BaseAppenderConfig'
+import { typeOfSchema } from '../../../types';
+import { FileAppender } from './FileAppender';
+
+function getSchema() {
+  return schema.object({
+    ...getBaseRawSchema(),
+    kind: schema.literal('file'),
+    path: schema.string()
+  });
+}
+
+const schemaType = typeOfSchema(getSchema);
+type FileAppenderConfigSchemaType = typeof schemaType;
+
+export class FileAppenderConfig extends BaseAppenderConfig {
+  readonly path: string;
+
+  constructor(schema: FileAppenderConfigSchemaType) {
+    super(schema);
+
+    this.path = schema.path;
+  }
+
+  createAppender(): FileAppender {
```

Comment:
I have mixed feelings too, let's discuss in zoom. The idea was that plugin just registers new config schema with our logging subsystem and that's it, the new appenders that plugin provides are created based on it. But I'm sure there is a better way to do this :D